client/src/components/user/ApiKey.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from 'react'
+import { Card, CardText } from 'material-ui/Card'
+import Button from 'material-ui/RaisedButton'
+import FlatButton from 'material-ui/FlatButton'
+import FontIcon from 'material-ui/FontIcon'
+import IconButton from 'material-ui/IconButton'
+import SubHeader from '../header/SubHeader'
+import { useQuery, deleteApiKey, generateApiKey } from '../../api/api'
+import NoteBlock from '../utils/NoteBlock'
+import Spinner from '../utils/Spinner'
+import copy from 'copy-text-to-clipboard'
+
+const requestOpts = {
+    useAuth: true,
+}
+
+const styles = {
+    flexCenterDiv: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        marginTop: '5px',
+    },
+}
+
+const ApiKeyView = () => {
+    return (
+        <div>
+            <SubHeader title="API key"></SubHeader>
+            <Card>
+                <CardText>
+                    <p>
+                        An API key can be used to upload new app versions
+                        through the App Hub API. This can be useful for use in
+                        continous integration workflows.
+                    </p>
+                    <ApiKeyStatus />
+                </CardText>
+            </Card>
+        </div>
+    )
+}
+
+const GenerateApiKey = props => {
+    return (
+        <div>
+            <NoteBlock>
+                Once an API-key is generated, you will not be able to view it
```

Comment:
To emphasise the 'treat API keys as passwords' part, I think we can remove the first sentence (users see it after generating an API key anyway).

Also a more warning-like colour (orange/red) would make this stand out more:
![image](https://user-images.githubusercontent.com/4295266/111652909-5d7fd100-87ff-11eb-82b4-b58e4aa2c0fd.png)